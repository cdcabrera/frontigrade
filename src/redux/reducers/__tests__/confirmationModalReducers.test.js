import { confirmationModalReducers } from '../';

describe('ConfirmationModalReducers', () => {
  it('should return the initial state', () => {
    expect(confirmationModalReducers.initialState).toBeDefined();
  });
});
