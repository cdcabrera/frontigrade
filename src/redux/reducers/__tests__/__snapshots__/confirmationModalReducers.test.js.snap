// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ConfirmationModalReducers should handle all defined types: defined types 1`] = `
Object {
  "result": Object {
    "show": true,
    "type": "CONFIRMATION_MODAL_SHOW",
  },
  "type": "CONFIRMATION_MODAL_SHOW",
}
`;

exports[`ConfirmationModalReducers should handle all defined types: defined types 2`] = `
Object {
  "result": Object {
    "show": false,
  },
  "type": "CONFIRMATION_MODAL_HIDE",
}
`;
